src/utils.ts

```diff
@@ -0,0 +1,42 @@
+import { BrowserInfo } from './types';
+import path from 'path';
+import { FileExistsMethod } from './types/internal';
+
+export function addArrayValueByKey<T> (collection: Record<string, T[]>, key: string, value: T): void {
+    if (!collection[key])
+        collection[key] = [value];
+    else if (!collection[key].includes(value))
+        collection[key].push(value);
+};
+
+export function getShouldUploadLayoutTestingData (layoutTestingEnabled: boolean | undefined, browsers: (BrowserInfo & { testRunId: string })[]): boolean {
+    return !!layoutTestingEnabled && browsers?.length <= 1;
```

Comment:
Layout testing does not work in multiple browser mode. Because we would need different baselines for each browser, which is not even supported by the `devextreme-screenshot-comparer` as far as I know. This condition does not prevent concurrency mode support.

DevExtreme tests [run in concurrency mode](https://github.com/DevExpress/DevExtreme/blob/22_2/.github/workflows/testcafe_tests.yml#L139) yet layout testing works for them.